src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurDetailLineGroupForCollector.java

```diff
@@ -178,20 +260,56 @@ protected KualiDecimal calculateTotalAmountForLines(List<ConcurStandardAccountin
         return Optional.of(offsetEntry);
     }
 
+    protected Optional<OriginEntryFull> doNotBuildOffsetOriginEntry(

```

Comment:
This is confusing, since it just returns an empty option, why are there input parameters.